TypeCobol/Compiler/CodeElements/CodeElement.cs

```diff
@@ -169,6 +177,11 @@ public override bool Equals(object obj)
                    this.Text == codeElement.Text;
         }
 
+        public override int GetHashCode()
+        {
+            return _type.GetHashCode() * 11 + ConsumedTokens.GetHashCode();
```

Comment:
GetHashCode doesn't compare the same properties than Equals, is it ok?
Same remarks than for Equals:
I think if the CodeElement contains consumedTokens, then the first token should be enough to compare.
CodeElement cannot overlaps.

Reply:
done: Equals and GetHashCode use "Type" and "ConsumedTokens".
https://github.com/TypeCobolTeam/TypeCobol/pull/1555/commits/77c907c8055e95bf877861ca4745d1033758792c